airbyte-integrations/connectors/source-jira/metadata.yaml

```diff
@@ -10,7 +10,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: 68e63de2-bb83-4c7e-93fa-a8a9051e3993
-  dockerImageTag: 3.0.11
+  dockerImageTag: 3.0.12
```

Comment:
Based on [this discussion](https://airbytehq-team.slack.com/archives/C02U9R3AF37/p1722355757502729), we should decide which version we want to set here